### DISCUSSION TITLE
StorageAccount - LargeFileSharesState

### DISCUSSION THREAD

**Discussion by dibcar** [Azure/ResourceModules#2568] [Q&A] [answered]
Hi, the property 'LargeFileSharesState' is not in the storageaccount module. Is there another way to set this property to 'enable' or the property is just missing in the module ?
Thanks

**Comment by eriqua** [comment, marked as answer]
Hello @dibcar, it looks like the property is missing from the module. I'll create an issue out of the discussion in order to triage it internally. Thanks for raising this!

**Reply by eriqua** [reply]
@dibcar issue #2570 will track the status going forward

**Reply by eriqua** [reply]
@dibcar the issue has been resolved and you can check the updated module version in the main branch.
